analysis_runner/cromwell.py

```diff
@@ -386,6 +387,29 @@ def _get_wait_interval(
                 continue
             status = r.json().get('status')
             _remaining_exceptions = max_sequential_exception_count
+
+            # if workflow has concluded print logging to hail batch log
+            # only do once workflow has reached a final state so as not
+            # to duplicate
+            if status.lower() in terminal_statuses:
+                logger.info('Cromwell workflow has concluded - fetching log')
+                metadata_url = (
```

Comment:
This file could be enormous (like, 50MB enormous), so printing it probably isn't a good idea. I'm generally happy for the download to occur (or even better, place in the output_path called like `<workflow_id>-metadata.json`), but it would be handy to print the visualisation defined here: https://github.com/populationgenomics/analysis-runner/blob/eecb4018255c6b5db429b1b314972b5fd0fe64ab/analysis_runner/cli_cromwell.py#L289-L294